test/test-integration-3rd/src/test/java/com/alipay/sofa/rpc/base/BaseZkTest.java

```diff
@@ -59,4 +60,30 @@ public static void adAfterClass() {
             }
         }
     }
+
+    /**
+     * 循环获取某个值，直到满足预期或者超时为止，相比sleep可高效
+     * @param callable 任务
+     * @param expect 期望值
+     * @param period 周期
+     * @param times 次数
+     * @param <T> 值类型
+     * @return 返回值
+     */
+    protected <T> T delayGet(Callable<T> callable, T expect, int period, int times) {
```

Comment:
At #180, I add `TestUtils.java` contains this method.  Just use `TestUtils.delayGet()` instead of it.

Reply:
OK, I have fixed it.